### DISCUSSION TITLE
remote db management

### DISCUSSION THREAD

**Discussion by nnachtegael** [evertramos/server-automation#7] [Ideas]
Could it be possible to expose the db port ( port 3306 for mysql) to permit remote db management based for example on a (y/N) and a 'port number?' user input  when running new-site.sh ?

**Comment by evertramos** [comment]
Hello @nnachtegael,

Sure it is possible, but normally when this is required, might be better use some proxy for it or manually, once there might be multiple sites and databases, and we must have only one service open for each port, in that case, if we set port 3306 for wordpress db, we will have an error and firing new site. But this might be an option in new script to open db for remote access informing the url or even creating a db proxy. I will think of something in this matter.
